tests/json-test/src/state_test.rs

```diff
@@ -1,98 +1,151 @@
-
-    //         let engine = NullEngine::cita();
-    //         let mut config = BlockSysConfig::default();
-    //         config.quota_price = string_2_u256(test.transaction.gas_price.clone());
-    //         config.economical_model = EconomicalModel::Charge;
-    //         config.quota_price = U256::from(1);
-
-    //         let idx_gas = &postdata.indexes[&String::from("gas")];
-    //         let idx_value = &postdata.indexes[&String::from("value")];
-    //         let idx_data = &postdata.indexes[&String::from("data")];
-    //         let str_gas = test.transaction.gas_limit.clone()[*idx_gas].clone();
-    //         let str_value = test.transaction.value.clone()[*idx_value].clone();
-    //         let str_data = test.transaction.data.clone()[*idx_data].clone();
-
-    //         let mut proto_tx = ProtoTransaction::new();
-    //         proto_tx.set_data(string_2_bytes(str_data));
-    //         proto_tx.set_value(string_2_bytes(str_value));
-    //         proto_tx.set_nonce(test.transaction.nonce.clone());
-    //         proto_tx.set_quota(string_2_u256(str_gas).low_u64());
-    //         if !test.transaction.to.is_empty() {
-    //             proto_tx.set_to(test.transaction.to.clone());
-    //         }
-
-    //         let tx = Transaction::create(&proto_tx).unwrap();
-    //         let sender = secret_2_address(&test.transaction.secret_key);
-    //         let signed_transaction = tx.fake_sign(sender);
-
-    //         // Execute transactions
-    //         let result: ApplyResult = state.apply(&env_info, &engine, &signed_transaction, &config);
-    //         match result {
-    //             Ok(outcome) => {
-    //                 debug!("lalalal receipt error: {:?}", outcome.receipt.error);
-    //             }
-    //             _ => panic!("apply_transaction: There must be something wrong!"),
-    //         }
-
-    //         // check root hash
-    //         state.commit().unwrap();
-    //         let root = state.root();
-    //         debug!("state.root {}", root);
-    //         assert_eq!(*root, string_2_h256(postdata.hash));
-    //     }
-    // }
+            let mut evm_context = Context::default();
+            evm_context.block_quota_limit = string_2_u256(str_block_gas.clone());
+            evm_context.coin_base = data.env.current_coinbase;
+            evm_context.block_number = string_2_u256(data.env.current_number.clone()).low_u64();
+            evm_context.timestamp = string_2_u256(data.env.current_timestamp.clone()).low_u64();
+            evm_context.difficulty = string_2_u256(data.env.current_difficulty.clone());
+            evm_context.quota_used = U256::zero();
+            evm_context.last_hashes = Arc::new(vec![string_2_h256(str_prev_hash)]);
+
+            /*let mut cfg = Config::default();
+            cfg.check_balance = true;
+            cfg.check_nonce = false;
+            cfg.block_gas_limit = string_2_u256(str_block_gas.clone()).low_u64();*/
```

Comment:
Add some comments about why those line commented out. If useless, delete it.

It is useful to person who want to understand this test later.